android/engine/src/main/java/org/smartregister/fhircore/engine/domain/model/OverflowMenuItemConfig.kt

```diff
@@ -17,16 +17,29 @@
 package org.smartregister.fhircore.engine.domain.model
 
 import kotlinx.serialization.Serializable
+import org.smartregister.fhircore.engine.configuration.navigation.MenuIconConfig
+import org.smartregister.fhircore.engine.util.extension.interpolate
 
 @Serializable
 data class OverflowMenuItemConfig(
   val id: Int = 1,
   val title: String = "",
   val confirmAction: Boolean = false,
-  val titleColor: String? = null,
+  val icon: MenuIconConfig? = null,
+  val titleColor: String = "#000000",
```

Comment:
Can we use colors from the `colors.xml`

Reply:
Oh I see this is a `Kotlin` data class used in the configuration, we do not have access to the context here

Reply:
Sure